docs/content/tutorial/connect-cluster.md

```diff
@@ -41,10 +41,14 @@ All connections are over TLS. PGO provides its own certificate authority (CA) to
 
 ## Connect an Application
 
-For this tutorial, we are going to connect [Keycloak](https://www.keycloak.org/), an open source identity management application. Keycloak can be deployed on Kubernetes and is backed by a Postgres database. While we provide an [example of deploying Keycloak](https://github.com/CrunchyData/postgres-operator-examples/tree/main/kustomize/keycloak) in the [Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples) repository, we will use the sample manifest below to deploy the application:
+For this tutorial, we are going to connect [Keycloak](https://www.keycloak.org/), an open source
+identity management application. Keycloak can be deployed on Kubernetes and is backed by a Postgres
+database. While we provide an [example of deploying Keycloak](https://github.com/CrunchyData/postgres-operator-examples/tree/main/kustomize/keycloak)
+in the [Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples)
+repository, we will use the manifest below to deploy the application using our `hippo` cluster:
```

Comment:
```suggestion
database. While we provide an [example of deploying Keycloak and a PostgresCluster](https://github.com/CrunchyData/postgres-operator-examples/tree/main/kustomize/keycloak)
in the [Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples)
repository, the manifest below deploys it using our `hippo` cluster that is already running:
```

Reply:
I like the suggestion :+1: 

Think that def makes it even clearer.